script/bootstrap-st2express

```diff
@@ -3,6 +3,10 @@
 
 PROJECT_ROOT=/opt/puppet
 
+if [ $UID -ne 0 ]; then
```

Comment:
I actually changed it to be POSIX compat. https://github.com/StackStorm/st2workroom/commit/24243c6e873a173d72f20c1e6f4c9f000e6cc101